sysdig/internal/client/v2/model.go

```diff
@@ -777,3 +777,30 @@ type SilenceRule struct {
 	Version int `json:"version,omitempty"`
 	ID      int `json:"id,omitempty"`
 }
+
+type CloudAccountSecure2 struct {
+	// TODO: Temporarily adding this to support till we add CloudAccountSecure here
+}
+
+type Timestamp struct {
+	Seconds int64 `json:"seconds,omitempty"`
+	Nanos   int32 `json:"nanos,omitempty"`
+}
+
+type OrganizationSecure struct {
```

Comment:
Can we just use the [actual type](https://github.com/draios/protorepo/blob/master/cloudauth/go/cloud_organization.pb.go#L24-L39) here? 

if we need to wrap it, or embed it, that still seems better than redefining it.

Reply:
Yes, I've been thinking about it too, if we can import and use the Cloud Account and Cloud Org object directly from `draios/protorepo/cloudauth`?
And then the `*FromResourceData` and `*ToResourceData` in resource*.go can do the from/to conversion of the fields as required.

Reply:
The only thing i am not sure about is that, this is a public repo, can we refer internal proto definitions in this? Any implications of exposing the proto to public?  
Well, the api is public facing too, so maybe that's not an issue.